docs/platform/howto/byoc/enable-byoc.md

```diff
@@ -26,56 +25,43 @@ call, we make sure BYOC can address them, and we check your environment
 eligibility for the feature.
 
 :::important
-Before getting down to enabling BYOC, check the availability of the
-feature in
-[Who is eligible for BYOC](/docs/platform/concepts/byoc#eligible-for-byoc), make sure you understand all the
-[limitations](/docs/platform/howto/byoc/enable-byoc#byoc-enable-limitations),
-and meet all the
-[prerequisites](/docs/platform/howto/byoc/enable-byoc#byoc-enable-prerequisites).
+Before enabling BYOC, confirm your
+[eligibility](/docs/platform/concepts/byoc#eligible-for-byoc) and review the
+[limitations](/docs/platform/howto/byoc/enable-byoc#byoc-enable-limitations).
 :::
 
 ## Limitations {#byoc-enable-limitations}
 
--   You need at least the Priority tier of Aiven support services to be
-    eligible for activating BYOC.
--   BYOC is supported with the
-    [standard deployment](/docs/platform/concepts/byoc#byoc-deployment) model only.
--   Only organization's administrators can request enabling BYOC.
-
-## Prerequisites {#byoc-enable-prerequisites}
-
--   Administrator's role for your Aiven organization
--   Access to [Aiven Console](https://console.aiven.io/)
```

Comment:
See above comment and note that the prereqs were consolidated to avoid duplication and not require the user to go to three separate sections to see what they need to do to be eligible. 

The only thing I removed really was the access to the console part, but I think that it's a given if you are an admin. :)

Reply:
https://github.com/aiven/aiven-docs/blob/c9e890d896ccd16805a256219064f88d4cf4355f/docs/platform/concepts/byoc.md?plain=1#L55-L59